src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -43,7 +43,7 @@ class ReplicatedSharedTensor(metaclass=SyMPCTensor):
 
     # Used by the SyMPCTensor metaclass
     METHODS_FORWARD = {"numel", "t", "unsqueeze", "view", "sum", "clone"}
-    PROPERTIES_FORWARD = {"T"}
+    PROPERTIES_FORWARD = {"T", "shape"}
```

Comment:
We might need to also remove it from here. We could add it when we would need it.

Reply:
I need this because when generating a PRZS share, it should be the shape of RSTensor.